rest/context.go

```diff
@@ -325,6 +329,32 @@ func (ctx *gorillaRequestContext) NextURL() (string, error) {
 	return u.String(), nil
 }
 
+func (ctx *gorillaRequestContext) BuildURL(routeName, resourceID string) (string, error) {
+	r, ok := ctx.Request()
+	if !ok {
+		return "", fmt.Errorf("Unable to build URL for route name %q with resource ID %q: No request available.",
+			routeName, resourceID)
```

Comment:
Error messages shouldn't be capitalized or end with punctuation.

https://github.com/golang/go/wiki/CodeReviewComments#error-strings